docs/examples/operations_index.py

```diff
@@ -0,0 +1,8 @@
+doc(title=("Operations", "="),
+    options=":maxdepth: 2",
+    entries=[
+        "general/general_ops_index",
+        "image_processing/index",
+        "audio_processing/index",
+        "sequence_processing/index",
```

Comment:
```suggestion
        "general/general_ops_index",
        "image_processing/index",
        "audio_processing/index",
        "sequence_processing/index",
```
I'm confused, should it have `.rst` postfix or not?

Reply:
As I tried to convey in the documentation: if you have a path to  some index file like `general/general_ops_index` or `image_processing/index` which previously was expected to be an .rst file, now will expect an `.py` file with `doc()` statement and parse it recursively. 
If you want to end the Python parsing the new way of specifying the docs, you just add the full path with `.rst` extension, and use a regular rst file in that location.

Reply:
If you have any suggestion how to make it clearer in the docs, please share.

Reply:
Now I get it.
Maybe we should be explicit and add `.py` postfix and then `_document_examples` would check if the file is python indeed? Now you would get strange errors that the tool cannot open the `py` file even if we just specified `audio_processing/index`.

Reply:
Ok, I can adjust to explicit `.py` extensions and filter out on those.